Remove "interact" flags for tests that can be run automatically

mrego opened the pull request:
There were a bunch of tests that have both a reference file and
the "interact" flag.
They use some small JavaScript to execute some steps
before checking the output against reference file.
This caused them to be wrongly considered manual tests
when they can be run automatically.

For more info check w3c/wpt-tools#196 and w3c/wpt-tools#197.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/w3c/web-platform-tests/5370)
<!-- Reviewable:end -->

mrego committed:
Remove "interact" flags for tests that can be run automatically

There were a bunch of tests that have both a reference file and
the "interact" flag.
They use some small JavaScript to execute some steps
before checking the output against reference file.
This caused them to be wrongly considered manual tests
when they can be run automatically.

For more info check w3c/wpt-tools#196 and w3c/wpt-tools#197.

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/218642643)

# Firefox (nightly channel) #
Testing web-platform-tests at revision 0a3e7c7a88bc42ea1a0cad5f090983f925014bcb
Using browser at version BuildID 20170404100210; SourceStamp b5d8b27a753725c1de41ffae2e338798f3b5cacd
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>32 tests ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/CSS2/backgrounds/background-root-012.xht">/css/CSS2/backgrounds/background-root-012.xht</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/CSS2/backgrounds/background-root-013.xht">/css/CSS2/backgrounds/background-root-013.xht</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/CSS2/backgrounds/background-root-014.xht">/css/CSS2/backgrounds/background-root-014.xht</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/CSS2/backgrounds/background-root-017.xht">/css/CSS2/backgrounds/background-root-017.xht</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/css-regions-1/contentEditable/contentEditable-001.html">/css/css-regions-1/contentEditable/contentEditable-001.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/css-regions-1/contentEditable/contentEditable-002.html">/css/css-regions-1/contentEditable/contentEditable-002.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/css-regions-1/contentEditable/contentEditable-003.html">/css/css-regions-1/contentEditable/contentEditable-003.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/css-regions-1/contentEditable/contentEditable-004.html">/css/css-regions-1/contentEditable/contentEditable-004.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/css-regions-1/contentEditable/contentEditable-005.html">/css/css-regions-1/contentEditable/contentEditable-005.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/css-regions-1/contentEditable/contentEditable-006.html">/css/css-regions-1/contentEditable/contentEditable-006.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/css-regions-1/contentEditable/contentEditable-007.html">/css/css-regions-1/contentEditable/contentEditable-007.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/css-regions-1/contentEditable/contentEditable-008.html">/css/css-regions-1/contentEditable/contentEditable-008.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/css-regions-1/contentEditable/contentEditable-009.html">/css/css-regions-1/contentEditable/contentEditable-009.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/css-regions-1/contentEditable/contentEditable-010.html">/css/css-regions-1/contentEditable/contentEditable-010.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/css-regions-1/contentEditable/contentEditable-011.html">/css/css-regions-1/contentEditable/contentEditable-011.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/css-writing-modes-3/background-position-vrl-018.xht">/css/css-writing-modes-3/background-position-vrl-018.xht</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/css-writing-modes-3/background-position-vrl-020.xht">/css/css-writing-modes-3/background-position-vrl-020.xht</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/css-writing-modes-3/background-position-vrl-022.xht">/css/css-writing-modes-3/background-position-vrl-022.xht</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/selectors4/focus-within-001.html">/css/selectors4/focus-within-001.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/selectors4/focus-within-002.html">/css/selectors4/focus-within-002.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/selectors4/focus-within-003.html">/css/selectors4/focus-within-003.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/selectors4/focus-within-004.html">/css/selectors4/focus-within-004.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/selectors4/focus-within-005.html">/css/selectors4/focus-within-005.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/selectors4/focus-within-006.html">/css/selectors4/focus-within-006.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/selectors4/focus-within-shadow-001.html">/css/selectors4/focus-within-shadow-001.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/selectors4/focus-within-shadow-002.html">/css/selectors4/focus-within-shadow-002.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/selectors4/focus-within-shadow-003.html">/css/selectors4/focus-within-shadow-003.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/selectors4/focus-within-shadow-004.html">/css/selectors4/focus-within-shadow-004.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/selectors4/focus-within-shadow-005.html">/css/selectors4/focus-within-shadow-005.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/vendor-imports/mozilla/mozilla-central-reftests/selectors4/focus-within-1.html">/css/vendor-imports/mozilla/mozilla-central-reftests/selectors4/focus-within-1.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/vendor-imports/mozilla/mozilla-central-reftests/selectors4/focus-within-2.html">/css/vendor-imports/mozilla/mozilla-central-reftests/selectors4/focus-within-2.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5370/css/vendor-imports/mozilla/mozilla-central-reftests/selectors4/focus-within-3.html">/css/vendor-imports/mozilla/mozilla-central-reftests/selectors4/focus-within-3.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
</details>

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/218642644)

# Chrome (unstable channel) #
Testing web-platform-tests at revision 0a3e7c7a88bc42ea1a0cad5f090983f925014bcb
Using browser at version 59.0.3053.3 dev
Starting 10 test iterations

frivoal commented on css/selectors4/focus-within-001.html:
```diff
@@ -6,7 +6,6 @@
 <link rel="author" title="Florian Rivoal" href="mailto:florian@rivoal.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
 <link rel="match" href="focus-within-001-ref.html">
-<meta name="flags" content="interact">
```
* It is better to switch to `<meta name="flags" content="">` than to remove the line. Makes it clear that you have considered the question and decided that there was no need for flags, instead of maybe forgetting to think about it.

* If you remove the interact flag, the comment above the javascript about it being an optional convenience becomes wrong and needs to be removed: either the test is interactive and the js is a optional helper, or it is automated and js is required.

* As we switch to js required, it would be good to add `class="reftest-wait"` on the root, and remove it once js has run, so that we make sure that the ref comparison is done at the right time, and that if js fails to run for some reason, we detect it as a test that failed to run rather than a failure to implement focus-within correctly.

mrego replied:
Ok, I've done this in all the tests.

I'm still removing the empty `flags`, but I can add it if it's a blocker to land this. I have seen many tests that don't have it.

frivoal commented on css/selectors4/focus-within-002.html:
```diff
@@ -6,7 +6,6 @@
 <link rel="author" title="Florian Rivoal" href="mailto:florian@rivoal.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
 <link rel="match" href="focus-within-001-ref.html">
-<meta name="flags" content="interact">
```
same comment as for focus-within-001

frivoal commented on css/selectors4/focus-within-003.html:
```diff
@@ -6,7 +6,6 @@
 <link rel="author" title="Florian Rivoal" href="mailto:florian@rivoal.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
 <link rel="match" href="focus-within-001-ref.html">
-<meta name="flags" content="interact">
```
same comment as for focus-within-001

frivoal commented on css/selectors4/focus-within-004.html:
```diff
@@ -6,7 +6,6 @@
 <link rel="author" title="Florian Rivoal" href="mailto:florian@rivoal.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
 <link rel="match" href="focus-within-001-ref.html">
-<meta name="flags" content="interact">
```
same comment as for focus-within-001

frivoal commented on css/selectors4/focus-within-005.html:
```diff
@@ -6,7 +6,6 @@
 <link rel="author" title="Florian Rivoal" href="mailto:florian@rivoal.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
 <link rel="match" href="focus-within-001-ref.html">
-<meta name="flags" content="interact">
```
same comment as for focus-within-001

frivoal commented on css/selectors4/focus-within-006.html:
```diff
@@ -6,7 +6,6 @@
 <link rel="author" title="Florian Rivoal" href="mailto:florian@rivoal.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
 <link rel="match" href="focus-within-001-ref.html">
-<meta name="flags" content="interact">
```
same comment as for focus-within-001

frivoal commented on css/selectors4/focus-within-shadow-001.html:
```diff
@@ -6,7 +6,7 @@
 <link rel="author" title="Florian Rivoal" href="mailto:florian@rivoal.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
 <link rel="match" href="focus-within-shadow-001-ref.html">
-<meta name="flags" content="interact dom">
```
Similarly to the comment on focus-within-001:

As we switch to js required, it would be good to add `class="reftest-wait"` on the root, and remove it once js has run, so that we make sure that the ref comparison is done at the right time, and that if js fails to run for some reason, we detect it as a test that failed to run rather than a failure to implement focus-within correctly.

I note that this test already has that class on the `#focusme` element, but that's not where it is supposed to be, so that's a mistake that survived the previous review. Would be good to fix while we're at it.

mrego replied:
Removed the wrong "reftest-wait" class too.

frivoal commented on css/selectors4/focus-within-shadow-002.html:
```diff
@@ -6,7 +6,7 @@
 <link rel="author" title="Florian Rivoal" href="mailto:florian@rivoal.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
 <link rel="match" href="focus-within-shadow-001-ref.html">
-<meta name="flags" content="interact dom">
```
Same comment as on focus-within-shadow-001

frivoal commented on css/selectors4/focus-within-shadow-003.html:
```diff
@@ -6,7 +6,7 @@
 <link rel="author" title="Florian Rivoal" href="mailto:florian@rivoal.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
 <link rel="match" href="focus-within-shadow-001-ref.html">
-<meta name="flags" content="interact dom">
```
Same comment as on focus-within-shadow-001

frivoal commented on css/selectors4/focus-within-shadow-004.html:
```diff
@@ -6,7 +6,7 @@
 <link rel="author" title="Florian Rivoal" href="mailto:florian@rivoal.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
 <link rel="match" href="focus-within-shadow-001-ref.html">
-<meta name="flags" content="interact dom">
```
Same comment as on focus-within-shadow-001

frivoal commented on css/selectors4/focus-within-shadow-005.html:
```diff
@@ -5,7 +5,7 @@
 <link rel="author" title="Keyong Li" href="mailto:kli79@bloomberg.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
 <link rel="match" href="focus-within-shadow-001-ref.html">
-<meta name="flags" content="interact dom">
```
Same comment as on focus-within-shadow-001

frivoal commented on css/vendor-imports/mozilla/mozilla-central-reftests/selectors4/focus-within-1.html:
```diff
@@ -8,7 +8,6 @@
     <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
     <meta name="assert" content="Test checks :focus-within works after focus change">
     <link rel="match" href="focus-within-1-ref.html">
-    <meta name="flags" content="interact">
```
Similarly to focus-within-001:

*  It is better to switch to `<meta name="flags" content="">` than to remove the line. Makes it clear that you have considered the question and decided that there was no need for flags, instead of maybe forgetting to think about it.

* As we switch to js required, it would be good to add `class="reftest-wait"` on the root, and remove it once js has run, so that we make sure that the ref comparison is done at the right time, and that if js fails to run for some reason, we detect it as a test that failed to run rather than a failure to implement focus-within correctly.

frivoal commented on css/vendor-imports/mozilla/mozilla-central-reftests/selectors4/focus-within-2.html:
```diff
@@ -8,7 +8,6 @@
     <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
     <meta name="assert" content="Test checks :focus-within should not apply after element removed.">
     <link rel="match" href="focus-within-2-ref.html">
-    <meta name="flags" content="interact">
```
same comment as focus-within-1.html

frivoal commented on css/vendor-imports/mozilla/mozilla-central-reftests/selectors4/focus-within-3.html:
```diff
@@ -8,7 +8,6 @@
     <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
     <meta name="assert" content="Test checks :focus-within should not work for iframe">
     <link rel="match" href="focus-within-3-ref.html">
-    <meta name="flags" content="interact">
```
same comment as focus-within-1.html

frivoal commented on css/CSS2/backgrounds/background-root-012.xht:
```diff
@@ -7,7 +7,7 @@
   <link rel="help" href="http://www.w3.org/TR/CSS21/colors.html#background"/>
   <link rel="match" href="background-root-012-ref.xht" />
 
-  <meta name="flags" content="image interact"/>
```
(not my spec) This test seems to have two parts. one that matches the ref, and one where the user is invited to resize the window and observe effects. Only the first part is non interactive. I don't believe that removing the flag without modifying the test of splitting it in two would be correct.

mrego replied:
Yeah I've split them in 2, so we have the automatic and the manual part.

frivoal commented on css/CSS2/backgrounds/background-root-013.xht:
```diff
@@ -6,7 +6,7 @@
   <link rel="alternate" href="http://www.hixie.ch/tests/adhoc/css/background/10.html" type="text/html"/>
   <link rel="help" href="http://www.w3.org/TR/CSS21/colors.html#background"/>
   <link rel="match" href="background-root-013-ref.xht" />
-  <meta name="flags" content="image interact"/>
```
same comment as background-root-012.xht

frivoal commented on css/CSS2/backgrounds/background-root-014.xht:
```diff
@@ -6,7 +6,7 @@
   <link rel="alternate" href="http://www.hixie.ch/tests/adhoc/css/background/11.html" type="text/html"/>
   <link rel="help" href="http://www.w3.org/TR/CSS21/colors.html#background"/>
   <link rel="match" href="background-root-014-ref.xht" />
-  <meta name="flags" content="image interact"/>
```
same comment as background-root-012.xht

frivoal commented on css/CSS2/backgrounds/background-root-017.xht:
```diff
@@ -9,7 +9,6 @@
   <link rel="help" href="http://www.w3.org/TR/CSS21/colors.html#background"/>
   <link rel="match" href="background-root-017-ref.xht" />
 
-  <meta name="flags" content="interact"/>
```
Please use `<meta name="flags" content="" />` rather than deleting the line. See the comment on focus-within-001 for rationale.

frivoal commented on css/css-regions-1/contentEditable/contentEditable-001.html:
```diff
@@ -7,7 +7,7 @@
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#the-flow-into-property">
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#flow-from">
 		<link rel="help" href="http://www.w3.org/TR/html5/editing.html#contenteditable">
-		<meta name="flags" content="dom interact">
```
(not my spec) Similarly to the comment on focus-within-001:

As we switch to js required instead of js merely being a helper, it would be good to add class="reftest-wait" on the root, and remove it once js has run, so that we make sure that the ref comparison is done at the right time, and that if js fails to run for some reason, we detect it as a test that failed to run rather than a failure to implement focus-within correctly.

gsnedders replied:
There's nothing async in there, so it should be fine just taking the screenshot after the load event fires.

frivoal commented on css/css-regions-1/contentEditable/contentEditable-002.html:
```diff
@@ -7,7 +7,7 @@
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#the-flow-into-property">
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#flow-from">
 		<link rel="help" href="http://www.w3.org/TR/html5/editing.html#contenteditable">
-		<meta name="flags" content="dom interact">
```
same comment as css-regions-1/contentEditable/contentEditable-001.html

frivoal commented on css/css-regions-1/contentEditable/contentEditable-004.html:
```diff
@@ -8,7 +8,7 @@
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#the-flow-into-property">
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#flow-from">
 		<link rel="help" href="http://www.w3.org/TR/html5/editing.html#contenteditable">
-		<meta name="flags" content="dom interact">
```
same comment as css-regions-1/contentEditable/contentEditable-001.html

frivoal commented on css/css-regions-1/contentEditable/contentEditable-005.html:
```diff
@@ -7,7 +7,7 @@
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#the-flow-into-property">
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#flow-from">
 		<link rel="help" href="http://www.w3.org/TR/html5/editing.html#contenteditable">
-		<meta name="flags" content="dom interact">
```
same comment as css-regions-1/contentEditable/contentEditable-001.html

frivoal commented on css/css-regions-1/contentEditable/contentEditable-006.html:
```diff
@@ -8,7 +8,7 @@
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#the-flow-into-property">
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#flow-from">
 		<link rel="help" href="http://www.w3.org/TR/html5/editing.html#contenteditable">
-		<meta name="flags" content="dom interact">
```
same comment as css-regions-1/contentEditable/contentEditable-001.html

frivoal commented on css/css-regions-1/contentEditable/contentEditable-007.html:
```diff
@@ -8,7 +8,7 @@
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#the-flow-into-property">
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#flow-from">
 		<link rel="help" href="http://www.w3.org/TR/html5/editing.html#contenteditable">
-		<meta name="flags" content="dom interact">
```
same comment as css-regions-1/contentEditable/contentEditable-001.html

frivoal commented on css/css-regions-1/contentEditable/contentEditable-008.html:
```diff
@@ -7,7 +7,7 @@
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#the-flow-into-property">
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#flow-from">
 		<link rel="help" href="http://www.w3.org/TR/html5/editing.html#contenteditable">
-		<meta name="flags" content="dom interact">
```
same comment as css-regions-1/contentEditable/contentEditable-001.html

frivoal commented on css/css-regions-1/contentEditable/contentEditable-009.html:
```diff
@@ -7,7 +7,7 @@
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#the-flow-into-property">
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#flow-from">
 		<link rel="help" href="http://www.w3.org/TR/html5/editing.html#contenteditable">
-		<meta name="flags" content="dom interact">
```
same comment as css-regions-1/contentEditable/contentEditable-001.html

frivoal commented on css/css-regions-1/contentEditable/contentEditable-010.html:
```diff
@@ -7,7 +7,7 @@
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#the-flow-into-property">
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#flow-from">
 		<link rel="help" href="http://www.w3.org/TR/html5/editing.html#contenteditable">
-		<meta name="flags" content="dom interact">
```
same comment as css-regions-1/contentEditable/contentEditable-001.html

frivoal commented on css/css-regions-1/contentEditable/contentEditable-011.html:
```diff
@@ -7,7 +7,7 @@
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#the-flow-into-property">
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#flow-from">
 		<link rel="help" href="http://www.w3.org/TR/html5/editing.html#contenteditable">
-		<meta name="flags" content="dom interact">
```
same comment as css-regions-1/contentEditable/contentEditable-001.html

frivoal reviewed (changes requested):
I've reviewed this because some tests for focus within where in there, and I've worked on that feature and some of these tests. I've reviewed the rest of the PR while I was at it, but feedback from someone familiar with the tests / specs being tests would probably be good.

Please see the individual comments from adjustments that should be made before merging.

mrego commented:
Thanks @frivoal for the thorough review. I'll do the changes, but I'm not 100% sure about the empty `flags` stuff:
```html
  <meta name="flags" content="">
```

I cannot find any documentation saying that this is mandatory, and the [initial examples in the doc](http://web-platform-tests.org/writing-tests/reftests.html) don't use it.
@gsnedders is there any agreement regarding this topic?

mrego committed:
Apply suggested comments on review for patch that removes "interact" flag

Basically this patch does the following:
* Duplicate some tests so now they have an automatic part and a manual one
  (as it's required to resize the window).
* Use "reftest-wait" class to be sure that the JavaScript steps
  have been completed before checking the reference.
* Remove some wrong "reftest-wait" used outside the root element.
* Remove some wrong comments now that the tests are automatic.

w3c-bots commented:
These tests are now available on [w3c-test.org](https://w3c-test.org/submissions/5370/)

mrego commented:
@frivoal I've applied the suggested changes, expect keeping the empty `flags`, and uploaded a new version.

gsnedders commented:
Per the readme, we shouldn't be editing things in `css/vendor-import`. That said, I've got no idea if @dbaron is going to be importing any upstream changes any time soon, and how much of a pain exporting changes here in the meantime will be?

gsnedders commented:
@mrego I don't think there's any real preference either way. I'd rather not have them bloating the file, on the whole.

frivoal commented:
@mrego: based on @gsnedders's comment, can you back out the changes to the tests under `vendor-imports/` These are meant to be fixed upstream, not here. Let's not cause any merge conflict for next time Mozilla wants to sync things. Maybe you want to send this fix directly to them? I don't know if you have access to Mozilla's repos.

Once that's done, I'll merge.

mrego committed:
Do not modify tests under css/vendor-imports/

mrego commented:
Ok, I've just reverted the changes on the Mozilla tests, I'll send them upstream to Firefox bugzilla.

frivoal commented:
@mrego Having some problems with travis failing tests. Can you rebase on top of the latest master to catch the fix for #5330 ?

frivoal commented:
@mrego actually, I see that you've already merged master in. @gsnedders, any idea what travis is still complaining about?

gsnedders commented:
Probably https://github.com/jugglinmike/chrome-screenshot-race/issues/1.

gsnedders commented:
@frivoal the failure mode of #5330 is getting the "Unstable results" with nothing below it, for reference.

mrego commented:
I'm not sure if I'm checking the right thing, but the [results I see](https://travis-ci.org/w3c/web-platform-tests/jobs/218931179) don't make a lot of sense to me.
Specially the `focus-within` ones:
```
ERROR:check_stability:| `/css/selectors4/focus-within-001.html`                       |         | **FAIL: 6/10, PASS: 4/10** |          |
```
The feature is not implemented yet, so it's impossible that the test is passing.

frivoal commented:
@gsnedders @jugglinmike @mrego 

> The feature is not implemented yet, so it's impossible that the test is passing.

Right, the test results don't make sense to me either. Here and in other ref-tests PR I've been working on (#5285, #5283). If the Chrome stability test isn't working right due to https://github.com/jugglinmike/chrome-screenshot-race/issues/1 or whatever, shouldn't we disable it until it is fixed? For now it's doing more harm than good.

gsnedders commented:
The latest Travis build seems to have run into https://github.com/w3c/web-platform-tests/issues/5407

mrego commented:
JFTR, Mozilla tests have been modified upstream: https://bugzilla.mozilla.org/show_bug.cgi?id=1354006

mrego commented:
Ok, it seems we can merge this now. Nice!
